main.go

```diff
@@ -231,6 +254,71 @@ func run(ctrlConfig spirev1alpha1.ControllerManagerConfig, options ctrl.Options)
 		return err
 	}
 
+	_, resources, err := clientset.ServerGroupsAndResources()
```

Comment:
We can query just for the spirev1alpha1 resources. Also, the error should be checked:
```suggestion
	resources, err := clientset.ServerResourcesForGroupVersion(spirev1alpha1.GroupVersion.String())
	if err != nil {
                ...
```